draft-ietf-tls-tls13.md

```diff
@@ -2221,6 +2226,25 @@ Note: In versions of TLS prior to TLS 1.3, this extension was named
 "elliptic_curves" and only contained elliptic curve groups. See
 {{RFC4492}} and {{I-D.ietf-tls-negotiated-ff-dhe}}.
 
+All clients MUST send a valid "supported_groups" extension containing
+at least one group for each key exchange algorithm (currently
+DHE and ECDHE) for which it offers a cipher suite.
```

Comment:
How so? It's replacing this, from down below in this same section:
> The client MUST supply a "supported_groups" extension containing at
> least one group for each key exchange algorithm (currently
> DHE and ECDHE) for which it offers a cipher suite.

It's almost entirely a cut/paste from your original text.

I don't see anything that PSK needs special note for.